libafl/src/events/launcher.rs

```diff
@@ -461,7 +461,7 @@ where
                                 if self.stdout_file.is_some() || self.stderr_file.is_some() {
                                     stdout = Stdio::inherit();
                                     stderr = Stdio::inherit();
-                                };
+                                }
```

Comment:
clippy was not happy

Reply:
unrelated change, should I keep it nevertheless?

Reply:
Keep what?